src/Authorization/appsettings.json

```diff
@@ -43,5 +43,12 @@
     "ApiRegisterEndpoint": "http://localhost:5101/register/api/v1/",
     "ApiStorageEndpoint": "http://localhost:5010/storage/api/v1/",
     "ApiResourceRegistryEndpoint": "http://localhost:5101/resourceregistry/api/v1/"
+  },
+  "QueueStorageSettings": {
+    "EventLogConnectionString": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1",
+    "AuthorizationEventQueueName": "authorizationeventlog"
+  },
+  "FeatureManagement": {
+    "AuditLog": true
```

Comment:
Should default value in appsettings.json not be false for the feature?
And force activation through values.yaml per environment?

Reply:
yeah, that was comitted by mistake. I will fix it